docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -14,7 +14,7 @@ When writing applications for the ledger in Java, you want to work with a repres
 Download
 ========
 
-You can download the `latest version <https://bintray.com/api/v1/content/digitalassetsdk/DigitalAssetSDK/com/daml/java/codegen/$latest/codegen-$latest.jar?bt_package=sdk-components>`__  of the Java codegen.
+You can download the `latest version <https://bintray.com/api/v1/content/digitalassetsdk/DigitalAssetSDK/com/daml/java/codegen/$latest/codegen-$latest.jar?bt_package=sdk-components>`__  of the Java codegen. Make sure the Java codegen version is the same as the SDK version you are using (see :ref:`assistant-manual-managing-releases`).
```

Comment:
"Make sure the Java codegen version is the same as the SDK version" How do I actually check that in practise - ie, where do I look?

Reply:
So the combo to check is:
- downloaded java codegen jar: actively done by the user
- maven dependency in pom.xml: comes initially from the quickstart, which depends on the SDK version
- SDK version

Reply:
How about something like:

```
Make sure that the following SDK component versions are the same:
* the downloaded Java codegen jar file, eg. 10x.y.z
* the dependency to bindings-java, eg. 10x.y.z
* the sdk-version attribute in the da.yaml file, eg. x.y.z
```

Reply:
That looks perfect!